app/views/sep_history/_sep_history_display.html.erb

```diff
@@ -0,0 +1,90 @@
+<% @family.special_enrollment_periods.to_a.reverse_each do |f| %>
+  <div class="add-default-class-<%= f.id.to_s %>">
+    <div class="panel-heading sep-heading-<%= f.id.to_s %>" role="tab">
+      <div class="row">
+        <div class="col-md-8">
+          <a role="button" class='btn-nav-accordion collapsed' data-toggle="collapse" data-target="#viewSepPanel_<%= f.id.to_s %>" aria-expanded="false" style="color: black;" onclick='addStyleToPanel("<%= f.id.to_s %>")' >
+            <i class="fas fa-chevron-down"></i>
+            <%= f.title %>
+          </a>
+        </div>
+        <div class="col-md-2">
+          <%= f.start_on %>
+        </div>
+        <div class="col-md-2">
+          <%= f.end_on %>
+        </div>
+      </div>
+    </div>
+    <div id="viewSepPanel_<%= f.id.to_s %>" class="panel-collapse collapse" aria-expanded="false">
+      <div class="panel-body">
+        <div class="row mt-1">
+          <div class="col-md-6">
+            <div class="col-md-6 font-weight-bold"><%= l10n("hbx_profiles.sep_effective_date_rule") %></div>
+            <div class="col-md-6">
+              <% if f.effective_on_kind.present? %>
+                <% if f.effective_on_kind == 'first_of_month' %>
+                  15th of month
+                <% elsif f.effective_on_kind == 'first_of_next_month' %>
+                  End of Month
+                <% else %>
+                  <%=f.effective_on_kind.humanize %>
+                <% end %>
+              <% else %>
+                N/A
+              <% end %>
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="col-md-6 font-weight-bold"><%= l10n("hbx_profiles.sep_event_date") %></div>
+            <div class="col-md-6"><%= f.qle_on %></div>
+          </div>
+        </div>
+        <div class="row mt-1">
+          <div class="col-md-6">
+            <div class="col-md-6 font-weight-bold"><%= l10n("hbx_profiles.sep_creator") %></div>
+            <div class="col-md-6"><%= f.user_email.present? ? f.user_email : 'N/A' %></div>
```

Comment:
<img width="1083" alt="Screen Shot 2021-05-26 at 2 20 57 PM" src="https://user-images.githubusercontent.com/14282458/119711603-b6866980-be2d-11eb-9cd8-8803b98e2982.png">

By default it is displaying current user whoever logs.

Reply:
I think having email makes more sense to display, or we need to display hbx_id